.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2.0
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run:
+          name: download and untar files
+          command: |
+            ./.utils/prepare_files.sh
```

Comment:
When is that job run? We should not run it automatically on a fixed schedule, but only when we request a pull manually. That is to not interfere with running performance tests, and resulting different results due to changed media files.

Reply:
So do you mean that "pulling and installing the media files" should be initiated manually, not when there are commits into the "raptor" branch?

To you question, currently this step runs when something commits into the raptor branch.

Reply:
On this repository we do not have any media files. Reason is that those are way too large, and also don't change (maybe at all).

If new tests are added which require new media files, those would have to be downloaded first, and only then this repository has to be checked for updates. So best would be to have it manually, or?

Also please note that the target website is used frequently by automated tests. So updating the media files directly at the target location could cause test failures in Taskcluster. Maybe media files should first be downloaded to a separate folder, and then replaced?

Reply:
Ok, that sounds good. We'll update the media files manually whenever it's needed.

I updated this PR so that CircleCI only deploy "code" not "media".

Since updating the media files is going to be very rare, I think it can be easily coordinated when to run it. Let me know it must be an atom operation though.

Reply:
Should we create some documentation for how to update the media files?

Reply:
Sure thing. I've updated the PR with a README file.

Reply:
I would suggest that we also add this to the wiki of this repository, and maybe add a link inside the README. Please see https://github.com/mozilla/perf-youtube-playback/wiki